app/src/main/java/com/hedvig/app/ApplicationModule.kt

```diff
@@ -476,3 +478,10 @@ val useCaseModule = module {
     single { StartNorwegianAuthUseCase(get()) }
     single { SubscribeToAuthStatusUseCase(get()) }
 }
+
+val offerPersistenceManagerModule = module {
+    single {
```

Comment:
Should we explicitly mention that we're creating a `single<OfferPersistenceManager>`?

It honestly might not be necessary if this code runs without any issues, I was under the impression that it was needed though with Koin 🤔